dm/tests/tls/conf/generate_tls.sh

```diff
@@ -16,12 +16,12 @@ DNS.1 = localhost
 IP.1 = 127.0.0.1
 EOF
 
-openssl ecparam -out "ca.key" -name prime256v1 -genkey
+openssl genrsa -out "ca.key"
```

Comment:
refer: https://docs.pingcap.com/tidb/stable/generate-self-signed-certificates#issue-certificates-for-clients